openlibrary/plugins/openlibrary/js/index.js

```diff
@@ -109,4 +106,13 @@ jQuery(function () {
         import(/* webpackChunkName: "editions-table" */ './editions-table')
             .then(module => module.initEditionsTable());
     }
+    // Enable any carousels in the page:
```

Comment:
I'm not crazy about the light wrapper; can we remove it? If we do:


```js
const carousels = document.getElementsByClassName('carousel--progressively-enhanced');
if (carousels.length) {
    import(/* webpackChunkName: "carousels" */ './carousels')
    .then(module => module.initCarousels(carousels));
}
```

then in carousel/index.js
```js
import Carousel from './carouel.js'
export function initCarousel(carousels) {
            carousels.forEach(el => Carousel.add(...JSON.parse(el.dataset.config));
}
```

Then we 1) remove the duplicating lightweight wrapper (DRY), 2) follow the pattern set with `editions-table` (which will make it easier for future devs to use), and 3) keep `index.js` as a central hub for dynamic imports. Is this possible?

Reply:
done

Reply:
I forgot to mention - removing the light wrapper means that any cached HTML e.g. homepage will need to be refreshed as the Carousel object will now no longer be globally accessible.